### PR TITLE
[PD-1403] Parse query params as unicode instead of ascii.

### PR DESCRIPTION
**Pull request by aspidites**
Changed what's passed to urlencode to prevent errors.

### CODE REVIEW

File: universal/helpers.py [hunk 1]
```diff
@@ -23,11 +23,14 @@ def update_url_param(url, param, new_val):
     outputs:
     The new url.
     """
+
     url_parts = list(urlparse(url))
     parts = copy(url_parts)
-    query = dict(parse_qsl(parts[4]))
+    query = {k.encode('utf-8'): v.encode('utf-8')
+             for k, v in parse_qsl(parts[4])}
```
Comment (JLMcLaugh) [inline universal/helpers.py]: Could you write a regression test for this?

Comment (aspidites) [inline universal/helpers.py]: Yeah.

**Comment by aspidites** [comment]
@JLMcLaugh 
Hmmm... I have to remember how to reproduce this before I can write a regression test. I tried running unicode through the helper on qc and master vs this one and don't get errors, so it seems to be something in whatever uses this helper. Unfortunately, I'm having a heck of a time getting saved searches to send locally on staging. In other words, it could be a minute before I'm able to do as you ask. I'm not ignoring you.

**Comment by aspidites** [comment]
OK. Finally able to reproduce this.

**Comment by aspidites** [comment]
May already be fixed on QC. Closing this in the mean time.

**Comment by aspidites** [comment]
Correctly. Apparently if I am the recipient of the saved search, QC works, if @almathew is, it breaks. This PR fixes both cases though.

**Comment by fezick** [comment]
Did regression tests get written for this? I don't see it in the commit.

**Comment by aspidites** [comment]
@fezick no. Walking over to explain why...

**Comment by aspidites** [comment]
Figured it out (kind of). Writing a unit test now.

**Comment by aspidites** [comment]
@fezick @JLMcLaugh all: There's now a test for this case.

**Pull request merged**